src/components-examples/material/dialog/dialog-harness/dialog-harness-example.spec.ts

```diff
@@ -27,6 +28,8 @@ describe('DialogHarnessExample', () => {
     }).compileComponents();
     fixture = TestBed.createComponent(DialogHarnessExample);
     fixture.detectChanges();
+    fixtureTwo = TestBed.createComponent(DialogHarnessExample);
+    fixtureTwo.detectChanges();
```

Comment:
I don't follow why we need to duplicate the fixture.

Reply:
If you open a dialog with the same component immediately after the first was opened, without waiting 300ms, then it will throw an error.